.github/workflows/tests.yml

```diff
@@ -68,11 +75,34 @@ jobs:
           # Wait for OSDs to become up
           sleep 30
 
+      - name: Enable RGW
+        run: |
+          set -ux
+          sudo microceph enable rgw
+          # Wait for RGW to settle
+          for i in $(seq 1 8); do
+              res=$( ( sudo microceph.ceph status | grep -cF "rgw: 1 daemon" ) || true )
+              if [[ $res -gt 0 ]] ; then
+                  echo "Found rgw daemon"
+                  break
+              else
+                  echo -n '.'
+                  sleep 5
+              fi
+          done
+
+      - name: Run system tests
+        run: |
+          set -eux
           # Show ceph's status
           sudo microceph.ceph status
 
-          # Confirm ceph is healthy
+          # Confirm ceph is healthy and services started
           sudo microceph.ceph health | grep -xF "HEALTH_OK"
+          sudo microceph.ceph status | grep -F "mon: 1 daemons"
+          sudo microceph.ceph status | grep -E "mgr: .*active, "
+          sudo microceph.ceph status | grep -F "osd: 3 osds"
+          sudo microceph.ceph status | grep -F "rgw: 1 daemon"
```

Comment:
Can you split the changes to this file into a separate commit (same PR is fine)?